openflighthpc/carbon-leaderboard
# Leaderboard Styles Enhancement & Bug Fixes

# Overview

This PR aims to improve the leaderboard style so that it can have a responsive size with the window size changing. Besides, it adds national flag icons to the location column of the entries of the leaderboard. It also fixed a backend bug in `device_controller#raw_data` that causes an error when fetching the data without having any entries in the database.

# Testing

- Run `rails s` to start the local server.
- Access the leaderboard page `http://localhost:3000/leaderboard/grouped` through a browser.
- Resize the browser window to see the responsive resizing of the leaderboard.
